sdk/cognitivelanguage/Azure.AI.Language.Conversations/samples/Sample2_AnalyzeConversation_OrchestrationPrediction.md

```diff
@@ -28,44 +28,28 @@ Once you have created a client, you can call synchronous or asynchronous methods
 ```C# Snippet:ConversationAnalysis_AnalyzeConversationOrchestrationPrediction
 string projectName = "DomainOrchestrator";
 string deploymentName = "production";
-
-var data = new
-{
-    AnalysisInput = new
-    {
-        ConversationItem = new
-        {
-            Text = "How are you?",
-            Id = "1",
-            ParticipantId = "1",
-        }
-    },
-    Parameters = new
+AnalyzeConversationInput data = new ConversationalInput(
+    new ConversationAnalysisInput(
+        new TextConversationItem(
+            id: "1",
+            participantId: "1",
```

Comment:
Nit - might be good to have ParticipantA or something instead of "1", users may think that `id` and `pariticpantId` must match or be related if the values happen to look the same on docs

Reply:
For CL/Orchestration, particopant ID is not relevant. It should be "optional" but our swagger declares it as required. I kept as is with previous readme.

Reply:
But let me change.